src/Home.js

```diff
@@ -2,16 +2,27 @@ import React from 'react';
 import ModulePlan from './ModulePlan';
 import PlanningSection from './PlanningSection';
 import moduleplan from './moduleplan.json';
+import users from './users.json';
 
 class Home extends React.Component {
   render() {
```

Comment:
I put this review comment here because it applies to the whole `render` method.

`render` grew since the last Pull Request. The longer a method gets the harder it will get over time to understand what it does. To prevent that my suggestion is to split up `render` like this:

```javascript
render() {
  var semesters = this.getSemestersForUser();
  return (
    <div>
      <ModulePlan semesters={semesters}/>
      <PlanningSection />
    </div>
  );
}
```

`getSemestersForUser` then contains all the code from `render` which prepares the data (like getting only the modules from the `json` file which shall be displayed).

The advantage here is that by reading `render` it is clear what happens:
- First get data (call `getSemestersForUser`)
- Then build up the necessary HTML structure

If the reader is curious how the data is actually calculated (I know calculated is not the best word but I couldn't come up with a better one) they can browse into `getSemestersForUser` and have a look on the details.